client/src/components/helpers/spacer.tsx

```diff
@@ -10,13 +10,24 @@ const Padding = Object.freeze({
 type PaddingKeys = keyof typeof Padding;
 interface SpacerProps {
   size: PaddingKeys;
+  children?: JSX.Element;
+  direction?: 'top' | 'bottom';
 }
 
-const Spacer = ({ size }: SpacerProps): JSX.Element => (
+const Spacer = ({ size, children, direction }: SpacerProps): JSX.Element => (
   <div
     className='spacer'
-    style={{ padding: `${Padding[size]}px 0`, height: '1px' }}
-  />
+    style={{
+      padding:
+        direction === 'top'
+          ? `${Padding[size] * 2}px 0 0 0`
+          : direction === 'bottom'
+          ? `0 0 ${Padding[size] * 2}px 0`
+          : `${Padding[size]}px 0`
+    }}
```

Comment:
The padding is acting as `paddingBlock`, you don't need to introduce another padding direction for this issue.

```suggestion
      paddingBlock: `${Padding[size]}px 0`
    }}
```

the changes to the values should be made in the places, which can be wrapped in `<Spacer>`. You should calculate the needed new values, and add them to.

```ts
const Padding = Object.freeze({
  small: 5,
  medium: 15,
  large: 30,
  exLarge: 45
});
```

Reply:
hmmm if we are just wrapping the children without any padding directions, there wouldn't any to change the padding numbers or else it would alter how <Spacer/> works in other areas.
I'll take a look for places to use this version of Spacer tonight.